### DISCUSSION TITLE
CAPTCHA is not visible

### DISCUSSION THREAD

**Discussion by katastrophe92** [uBlockOrigin/uBlock-issues#2595] [Q&A]
Hello uBlock community! Thanks for this amazing add-on ❤ . 
How can I make CAPTCHA to work? Every time I register to something or make a online payment I must disable uBlock because the website maybe has a CAPTCHA and it is not showed. Do I need to make a rule to unblock it?
Hope someone can help me. Thanks!

**Comment by mapx-** [comment]
There is no general filter to help you.
Just report the page in trouble with all the necessary details to reproduce the issue.

**Comment by katastrophe92** [comment]
Hello @mapx- thanks for answering. I have 3rd-party scripts on, it might be this the problem?

**Reply by mapx-** [reply]
- if you do not provide links, details, steps to reproduce whatever issue you have, we cannot investigate your issues.
- use only the default settings in uBO, no dynamic filtering and test again

**Reply by Yuki2718** [reply]
> I have 3rd-party scripts on,

If it measn you generally blocked 3p scripts, CAPTCHA broken is just an expected result and you have to adjust by yourself, this is why dynamic filtering is behind "I am an advance user".

**Reply by krystian3w** [reply]
In Poland does not work pay by debit card / masterpass and visa wallets at "`cards.tpay.com`". For me, EasyPrivacy blocks something too aggressively there.

**Reply by mapx-** [reply]
Did you report the issue at EP repository ? @ryanbr

You have enough experience to deal with such issues.
What is blocked in uBO's logger ?
